e2e/tests/002.sh

```diff
@@ -37,3 +37,14 @@ k8s_wait_exists "$kubeconfig" 600 "default" "cluster" "edge02"
 
 k8s_wait_ready "$kubeconfig" 600 "default" "cluster" "edge01"
 k8s_wait_ready "$kubeconfig" 600 "default" "cluster" "edge02"
+
+# Inter-connect worker nodes
+kubectl get secret regional-kubeconfig -o jsonpath='{.data.value}' | base64 -d > $HOME/.kube/regional-kubeconfig
+kubectl get secret edge01-kubeconfig -o jsonpath='{.data.value}' | base64 -d > $HOME/.kube/edge01-kubeconfig
+kubectl get secret edge02-kubeconfig -o jsonpath='{.data.value}' | base64 -d > $HOME/.kube/edge02-kubeconfig
+export KUBECONFIG=$HOME/.kube/config:$HOME/.kube/regional-kubeconfig:$HOME/.kube/edge01-kubeconfig:$HOME/.kube/edge02-kubeconfig
+for context in $(kubectl config get-contexts --no-headers --output name); do
```

Comment:
This connects the mgmt cluster too. Is that what we want?

Reply:
The mgmt cluster is an [All-in-One cluster](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/bootstrap/tasks/main.yml#L33-L40), so technically it won't be connected.